src/main/java/com/softserve/borda/controllers/UserController.java

```diff
@@ -20,13 +23,19 @@
 //@SessionAttributes("user")
 public class UserController {
 
-    private final ModelMapper modelMapper;
+    @Autowired
+    private ModelMapper modelMapper;
 
-    private final UserService userService;
+    @Autowired
+    private UserService userService;
 
-    public UserController(UserService userService) {
-        this.userService = userService;
-        modelMapper = new ModelMapper();
+    private User user = null;
+
+    @ModelAttribute("user")
```

Comment:
@ModelAttribute annotation is for Spring MVC (https://www.baeldung.com/spring-mvc-and-the-modelattribute-annotation), it is not related to Rest API at all.